### ISSUE TITLE
Exclude folders from 'open resource'

### ISSUE THREAD

**Issue by markknol** [fdorg/flashdevelop#1024]
It would be nice if there was a setting to exclude folders from the 'open resource' panel.

**Comment by GimmickyApps** [comment]
For me, it would be even better if I could limit it completely to just my project files.  (If I need something from an external library my project references, I can usually just hit F4 to be brought there directly, right?)  Was that your use case too, or you need exclude folders in particular for something more refined?

**Comment by markknol** [comment]
@GimmickyApps  You mean this?
![image](https://cloud.githubusercontent.com/assets/576184/13808350/098b8be2-eb66-11e5-86af-54978e31f1b0.png)

**Comment by GimmickyApps** [comment]
That's what I had hoped those would do, but no.

"Code files only" is useful.  (I'm not sure what use it is without that checked, actually...because often for non-code files I want to right-click and edit, rather than open with the default viewer...e.g. audacity instead of vlc, gimp instead of windows preview).

"In Classpaths only" seems to get rid of my project's "export" dir (although, not entirely...there's still one at the bottom of the list, formerly there were three at the top), which is useful, but it still includes, for example, all of HaxeFlixel and so on.  So "main" gives me dozens of options, when I just want my project's main.  "Cat" gives anything with "appliCATion" in it, when I just want my project's Cat.hx class.

**Comment by Gama11** [comment]
Try [QuickNavigate](https://github.com/SlavaRa/fdplugin-quicknavigate)'s Type Explorer - much better suited to this use case (I still think it should be included in FD out of the box).

**Comment by GimmickyApps** [comment]
Oh my gosh, +1 for including it out of the box.  I've been missing something like Quick Outline ever since Visual Studio '97, and it's even better than what they had (drop-down box of methods, built into the toolbar.)  Thanks!

**Comment by GimmickyApps** [comment]
Although, unfortunately, Type Explorer then lacks the two options of Open Resource.  I wonder where one can find all three, just a quick way to open .hx files from your own project's source dir only, not export, not libs, not assets, not assetpaths classes.

**Comment by Gama11** [comment]
Have you made sure that "Search types in external classpaths" is unchecked?

**Comment by GimmickyApps** [comment]
Yes, and it still shows one class per asset (I will never need that) and stuff under my export directory.

**Comment by Gama11** [comment]
Ah, that. I think that's a FD issue since QuickNavigate just uses FD's classpaths, but not 100% sure. @SlavaRa?

**Comment by SlavaRa** [comment]
@Gama11 yes, that is true, QuickNavigate uses FD's classpaths.
